partials/contacts/detail.html

```diff
@@ -0,0 +1,11 @@
+<div>
+  <div class="col-xs-8 singleContact">
+    <button class="btn btn-default star" ng-if="!contact.isFavorite"><i class="fa fa-star-o" aria-hidden="true" ng-click="switchFavorite(contact, contact.id)"></i></button>
```

Comment:
This is pretty minor but the ng-click only works on the star now, not if I click on the button.